sections/main-password-footer.liquid

```diff
@@ -1,4 +1,5 @@
-{%- if section.settings.color_scheme == 'background-1' -%}<hr>{%- endif -%}
+{% assign scheme1 = settings.color_schemes | first %}
+{%- if section.settings.color_scheme == scheme1 -%}<hr>{%- endif -%}
```

Comment:
Why do we only show `hr` if it's `background-1`? Does that still work with the new schemes?

Reply:
Realized this code was actually for the password footer after I took these screenshots, but the answer is the same
<img src="https://screenshot.click/26-11-k3nfg-64gvh.png" width="300" />
<img src="https://screenshot.click/26-12-w5rcq-6qmek.png" width="300" />

Reply:
It was just because the color difference between schemes didn't require the line/border separation I believe. 
Seem to work as I'd expect but something a designer could confirm.